docs/index.md

```diff
@@ -30,7 +30,7 @@ This is a Web API reference application designed to "fork and code" with the fol
 
 ## Language Specific Documentation
 
-### [dotnet Implementation](CSharp-ASPDotNet-Developer.md)
+### [dotnet Implementation](dotnet-core-Developer.md)
 
```

Comment:
The rest of the md files in the docs folder are PascalCase -- we should keep consistent if possible

Reply:
changed to DotNet-Core-Developer.md in docs and file name